src/common/NetworkManager.ts

```diff
@@ -471,20 +409,22 @@ export class NetworkManager extends EventEmitter {
     const request = new HTTPRequest(
       this._client,
       frame,
-      interceptionId,
+      fetchRequestId,
       this._userRequestInterceptionEnabled,
       event,
       redirectChain
     );
-    this._requestIdToRequest.set(event.requestId, request);
+    this._networkRequestIdEventMap.httpRequest.set(event.requestId, request);
```

Comment:
should we rename _networkRequestIdEventMap and have a proper name?

Reply:
Sure. What name comes to mind?

Reply:
`_networkEventManager`?